### PR TITLE
[WFLY-19905] Move module org.apache.xml-resolver from WildFly Core to WildFly

### PR DESCRIPTION
**Pull request by parsharma**
JIRA: https://issues.redhat.com/browse/WFLY-11905

### CODE REVIEW

**Comment by bstansberry** [comment]
this is ok to test

File: ee-galleon-pack/pom.xml [hunk 1]
```diff
@@ -3978,6 +3978,11 @@
             </exclusions>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>xml-resolver</groupId>
+            <artifactId>xml-resolver</artifactId>
+ 	    <scope>provided</scope>
```
Comment (bstansberry) [inline ee-galleon-pack/pom.xml:3984]: There's something odd here with whitespace. When I use the View File option and see the whole file these line up. Perhaps there is a mix of tabs and spaces here?

File: pom.xml [hunk 2]
```diff
@@ -443,7 +444,7 @@
         <version.sun.jaxb>2.3.3-b02</version.sun.jaxb>
         <version.sun.saaj-impl>1.4.1.SP1</version.sun.saaj-impl>
         <version.wsdl4j>1.6.3</version.wsdl4j>
-        <version.xml-resolver>1.2</version.xml-resolver>
+	<version.xml-resolver>1.2</version.xml-resolver>
```
Comment (bstansberry) [inline pom.xml:447]: Please correct the whitespace.

File: pom.xml [hunk 3]
```diff
@@ -589,6 +590,12 @@
                 <version>${ee.maven.version}</version>
             </dependency>
 
+	   <dependency>
```
Comment (bstansberry) [inline pom.xml:593]: Please correct the whitespace.

**Review by bstansberry — CHANGES_REQUESTED** [review]
Thanks. I made some comments about whitespace but the work in general looks good!

File: ee-galleon-pack/pom.xml (hunk 1 above)
Comment (parsharma) [reply ee-galleon-pack/pom.xml:3984]: May be, I will check all the comments and update accordingly.

**Review by yersan — APPROVED** [review]
Changes look good to me once ident issues are resolved.

The Jira should be followed up by a removal of `org.apache.xml-resolver` on wildfly-core.

**Comment by bstansberry** [comment]
Thanks @parsharma .  Please squash these down to a single commit and add WFLY-19905 to the commit message and this is ready to merge.

**Pull request merged**